src/groupeddataframe/splitapplycombine.jl

```diff
@@ -27,7 +27,8 @@ function _combine_prepare(gd::GroupedDataFrame,
                           keepkeys::Bool, ungroup::Bool, copycols::Bool,
                           keeprows::Bool, renamecols::Bool)
     for cei in cs
-        @assert cei isa Union{Pair, Base.Callable, ColumnIndex, MultiColumnIndex}
+        @assert cei isa Union{Pair, Base.Callable, ColumnIndex, MultiColumnIndex,
+                              AbstractVecOrMat{<:Pair}}
```

Comment:
How were vectors supported before this PR?

Reply:
I could add `AbstractMatrix{<:Pair}` in the union, but I thought that it is cleaner to use `AbstractVecOrMat{<:Pair}` to signal the design intention.

Reply:
@nalimilan - do we agree on this approach?

Reply:
@nalimilan - ok to merge?